README.md

```diff
@@ -15,6 +15,12 @@
 # Nexus Repository Manger p2 Format
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-p2.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-p2%22) [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-p2/depshield.svg)](https://depshield.github.io)
 
+> **Huzzah!** p2 is now part of Nexus Repository Manager. Version 3.20 includes the p2 plugin by default. 
+>The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-p2](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-p2).
```

Comment:
The second link (nexus-repository-p2) 404s for me.  Is it TBD or is there something wrong (perms, mislink)?

Reply:
@joedragons it will be available after merging https://github.com/sonatype/nexus-internal/pull/5872